docs/widgets/select.md

```diff
@@ -57,6 +57,32 @@ The following example presents a `Select` with a number of options.
     --8<-- "docs/examples/widgets/select.tcss"
     ```
 
+## Example
```

Comment:
We don't want to have two H2 headings with the exact same name.
Instead, consider changing the original `## Example` to `## Examples` and then separate each example with an H3 heading, e.g., `### Basic Example` and `### Class method`.